examples/Sql/HiveDataFrame/Program.cs

```diff
@@ -24,7 +24,10 @@ static void Main(string[] args)
             var sparkConf = new SparkConf();
             var sparkContext = new SparkContext(sparkConf);
             var hiveContext = new HiveContext(sparkContext);
-            var peopleDataFrame = hiveContext.Read().Json(Path.Combine(Environment.CurrentDirectory, @"data\people.json"));
```

Comment:
I prefer to use "file:///usr/local/data/people.json" which corresponds to "hdfs://" prefix.